yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -90,12 +90,31 @@ def run(self, info):
 
         success = True
         if info['ext'] == 'mp3':
-            options = [
-                '-c', 'copy', '-map', '0:0', '-map', '1:0', '-write_id3v1', '1', '-id3v2_version', '3',
-                '-metadata:s:v', 'title="Album cover"', '-metadata:s:v', 'comment=Cover (front)']
+            # Method 1: Use mutagen
+            # Prioritize mutagen over ffmpeg since ffmpeg messes up the lyrics data
+            if mutagen:
+                try:
+                    self._report_run('mutagen', filename)
+                    audio = mutagen.id3.ID3(filename)
+                    with open(thumbnail_filename, 'rb') as thumbfile:
+                        audio['APIC'] = mutagen.id3.APIC(
+                            encoding=mutagen.id3.Encoding.UTF8, mime='image/%s' % thumbnail_ext, type=3,
+                            desc=u'Cover (front)', data=thumbfile.read())
```

Comment:
As per https://github.com/yt-dlp/yt-dlp/pull/8869#discussion_r1455878433. 
```suggestion
                        audio['APIC'] = mutagen.id3.APIC(
                            encoding=mutagen.id3.Encoding.UTF8, mime=f'image/{thumbnail_ext}',
                            type=mutagen.id3.PictureType.COVER_FRONT, desc='Cover (front)', thumbfile.read())
```